benchmarks/CMakeLists.txt

```diff
@@ -32,13 +68,6 @@ endif()
 set(BENCHMARK_ENABLE_DOXYGEN OFF)
 set(BENCHMARK_ENABLE_INSTALL OFF)
 set(BENCHMARK_ENABLE_TESTING OFF)
-set(BUILD_SHARED_LIBS OFF)

```

Comment:
Question: Can we set additional options for Google Benchmark here? I observe that when configuring, it emits verbose and scary-looking (but ultimately innocuous) messages for two of its POSIX detections:

```
-- Performing Test HAVE_GNU_POSIX_REGEX
-- Performing Test HAVE_GNU_POSIX_REGEX
-- Performing Test HAVE_GNU_POSIX_REGEX -- failed to compile: Change Dir: D:/GitHub/STL/out/benchmark/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~1/MIB055~1/2022/Preview/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_6c2ce && [1/2] Building CXX object CMakeFiles\cmTC_6c2ce.dir\gnu_posix_regex.cpp.obj
FAILED: CMakeFiles/cmTC_6c2ce.dir/gnu_posix_regex.cpp.obj
C:\PROGRA~1\MIB055~1\2022\Preview\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /EHsc /W4  /Zi /Ob0 /Od /RTC1 -MT /showIncludes /FoCMakeFiles\cmTC_6c2ce.dir\gnu_posix_regex.cpp.obj /FdCMakeFiles\cmTC_6c2ce.dir\ /FS -c D:\GitHub\STL\benchmarks\google-benchmark\cmake\gnu_posix_regex.cpp
D:\GitHub\STL\benchmarks\google-benchmark\cmake\gnu_posix_regex.cpp(1): fatal error C1083: Cannot open include file: 'gnuregex.h': No such file or directory
ninja: build stopped: subcommand failed.


-- Performing Test HAVE_POSIX_REGEX
-- Performing Test HAVE_POSIX_REGEX
-- Performing Test HAVE_POSIX_REGEX -- failed to compile: Change Dir: D:/GitHub/STL/out/benchmark/CMakeFiles/CMakeTmp

Run Build Command(s):C:/PROGRA~1/MIB055~1/2022/Preview/Common7/IDE/COMMON~1/MICROS~1/CMake/Ninja/ninja.exe cmTC_ab99d && [1/2] Building CXX object CMakeFiles\cmTC_ab99d.dir\posix_regex.cpp.obj
FAILED: CMakeFiles/cmTC_ab99d.dir/posix_regex.cpp.obj
C:\PROGRA~1\MIB055~1\2022\Preview\VC\Tools\MSVC\1434~1.319\bin\Hostx64\x64\cl.exe  /nologo /TP   /DWIN32 /D_WINDOWS /EHsc /W4  /Zi /Ob0 /Od /RTC1 -MT /showIncludes /FoCMakeFiles\cmTC_ab99d.dir\posix_regex.cpp.obj /FdCMakeFiles\cmTC_ab99d.dir\ /FS -c D:\GitHub\STL\benchmarks\google-benchmark\cmake\posix_regex.cpp
D:\GitHub\STL\benchmarks\google-benchmark\cmake\posix_regex.cpp(1): fatal error C1083: Cannot open include file: 'regex.h': No such file or directory
ninja: build stopped: subcommand failed.
```

Can/should we silence this by setting `HAVE_GNU_POSIX_REGEX` and `HAVE_POSIX_REGEX` to `OFF`? I tried adding

```
set(HAVE_GNU_POSIX_REGEX OFF)
set(HAVE_POSIX_REGEX OFF)
```

here, and it indeed silenced the obnoxious messages.